tlslite/utils/asn1parser.py

```diff
@@ -8,6 +8,36 @@
 from .codec import Parser
 
 
+class TLVType(object):
+    """
+    Class that represents the Type part of a TLV based encoding.
```

Comment:
it extracts the "class" and the "primitive" flag, so I'd say it is ASN.1/BER specific, not a generic TLV encoding.

in general, it would be good to have test cases for this class and this class added in separate commit from the the changes to `ocsp.py`